desc/grid.py

```diff
@@ -838,6 +848,10 @@ def _create_nodes(self, nodes):
         """
         nodes = jnp.atleast_2d(jnp.asarray(nodes)).reshape((-1, 3)).astype(float)
         # Do not alter nodes given by the user for custom grids.
+        # In particular, do not modulo nodes by 2pi or 2pi/(NFP/NFP_umbilic_factor).
+        # This may cause the surface_integrals() function to fail recognizing
+        # surfaces outside the interval [0, 2pi] as duplicates. However, most
+        # surface integral computations are done with LinearGrid anyway.
```

Comment:
this comment isn't relevant anymore, so you could remove it

Reply:
Removed